Makefile

```diff
@@ -1,6 +1,6 @@
 CXX = g++
-CXXFLAGS = -std=c++11 -Iinclude
-LDFLAGS = -lcurl
+CXXFLAGS = -std=c++11 -Iinclude -I"C:\tools\curl-8.15.0_4-win32-mingw\include" -DCURL_STATICLIB
```

Comment:
You are not supposed to edit the default configuration. If you're using windows. Read CONTRIBUTING.md for setup